AmericanMedicalAssociation/ama-style-guide-2
# EWL-9078 homepage hero styleguide

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

**Jira Ticket**
- [EWL-9078: Home Page Hero Module Styleguide](https://issues.ama-assn.org/browse/EWL-9078)
- [EWL-9080: Home Page Hero Video Component Styleguide](https://issues.ama-assn.org/browse/EWL-9080)

## Description
Adds Homepage Hero module and video stub mock ups to styleguide.


## To Test
- Run gulp serve and open styleguide.
- Select 'Homepage' under the Pages tab
- Confirm styling of Hero module component matches the following Zeplins:

[Desktop](https://zpl.io/bJqpz7J)

[Tablet]( https://zpl.io/bLOW9eB)

[Mobile](https://zpl.io/bzx5lEz)

## Visual Regressions
- N/A

## Relevant Screenshots/GIFs
![Screen Shot 2021-10-04 at 1 40 42 PM](https://user-images.githubusercontent.com/67962801/135906265-d6096866-701e-46d4-b6f4-67cee9df51b2.png)


## Remaining Tasks
- N/A


## Additional Notes
- N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
